src/main/java/ledger/user_interface/ui_controllers/component/AccountBalanceLabel.java

```diff
@@ -78,7 +78,7 @@ private void calculateBalanceForAccount() {
 
         String accountName = this.currentAccount.getName();
         if (accountName.length() > 20) accountName = accountName.substring(0, 17) + "...";
```

Comment:
You may want to double check this max length. There's nothing special about 20 characters, it was just approximately all that could fit in the old UI. It probably changed.